stumpy/floss.py

```diff
@@ -429,6 +429,7 @@ def __init__(
         self._n = self._T.shape[0]
         self._last_idx = self._n - self._m + 1  # Depends on the changing length of `T`
         self._n_appended = 0
+        self._T_isfinite = np.isfinite(core.rolling_window(self._T, m))
```

Comment:
Can you please rename this to `self._T_subseq_isfinite`? This way it's obvious that this is storing the booleans for each subsequence in T rather than the individual elements of T.